testing/frr/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributor: Arthur Jones <arthur.jones@riverbed.com>
+pkgname=frr
+pkgver=5.0
+pkgrel=0
+pkgdesc="Free Range Routing is a fork of quagga"
+url="https://frrouting.org/"
+arch="x86_64"
+license="GPL-2.0"
+depends="json-c c-ares ipsec-tools iproute2 python py-ipaddr bash"
+makedepends="ncurses-dev net-snmp-dev gawk texinfo perl
+    acct autoconf automake bash
+    binutils bison bsd-compat-headers build-base
+    c-ares c-ares-dev ca-certificates cryptsetup-libs curl
+    device-mapper-libs expat fakeroot flex fortify-headers gdbm
+    git gmp isl json-c-dev kmod lddtree libacl libatomic libattr
+    libblkid libburn libbz2 libc-dev libcap libcurl libedit libffi libgcc
+    libgomp libisoburn libisofs libltdl libressl libssh2
+    libstdc++ libtool libuuid linux-headers lzip lzo m4 make mkinitfs mpc1
+    mpfr3 mtools musl-dev ncurses-libs ncurses-terminfo ncurses-terminfo-base
+    patch pax-utils pcre perl pkgconf python2 python2-dev readline
+    readline-dev sqlite-libs squashfs-tools sudo tar texinfo xorriso xz-libs
+    py-sphinx"
+checkdepends="pytest py-setuptools"
+install="$pkgname.pre-install $pkgname.pre-deinstall $pkgname.post-deinstall"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-dbg $pkgname-openrc"
+source="https://github.com/FRRouting/frr/archive/$pkgname-$pkgver.tar.gz docker-start make-check-bypass-isisd-tlv.patch tests-isisd-built-sources.patch"
+
+builddir="$srcdir"/$pkgname-$pkgname-$pkgver
+
+_sbindir=/usr/lib/frr
+_sysconfdir=/etc/frr
+_libdir=/usr/lib
+_localstatedir=/var/run/frr
+_user=frr
+
+build() {
+	cd "$builddir"
+	./bootstrap.sh
+	./configure \
+		--prefix=/usr \
+		--sbindir=$_sbindir \
+		--sysconfdir=$_sysconfdir \
+		--libdir=$_libdir \
+		--localstatedir=$_localstatedir \
+		--enable-systemd=no \
+		--enable-vtysh \
+		--enable-multipath=64 \
+		--enable-vty-group=frrvty \
+		--enable-user=$_user \
```

Comment:
Change:
```
--enable-user=$_user \
--enable-group=$_user
```

To e.g. the following (it will only grap the first word of the string in case more are added in the future):
```
--enable-user=${pkgusers%% *} \
--enable-group=${pkggroups%% *}
```

Reply:
Thank you!  Will fix...